lib/Logger.js

```diff
@@ -77,13 +86,15 @@ class Logger {
         const rLog = new RequestLogger(Config.logger,
                                        Config.level, Config.dump, Config.end,
                                        unserializeUids(serializedUids));
-        rLog.addDefaultFields({ name: this.name });
+        rLog.setParent(this);
         return rLog;
     }
 
     _doLog(levelName, msg, data) {
         const sLogger = Config.logger;
-        const finalData = { name: this.name, time: Date.now() };
+        const finalData = Object.assign({},
```

Comment:
Good point about using objectCopy.
As for computing the finalData, I need to compute it at the logging time, whether it's visible or not, since the fields may change in the meantime. There would be no meaning in logging fields "more recent" than the log entry itself. As there is no way to tie "default fields" and "log entry" in the timeline, this is the only option available to us.